src/dali/plugin/input/rest/coinbase.py

```diff
@@ -632,8 +633,11 @@ def _process_transfer(
                         # Incoming money from another Coinbase user. Marking it as income conservatively, but it could be
                         # a gift or other type: if so the user needs to explicitly recast it with a transaction hint
                         self._process_gain(transaction, currency, Keyword.INCOME, in_transaction_list, f"From: {transaction[_FROM][_EMAIL]}")
-                    elif transaction[_DETAILS][_SUBTITLE].startswith("From Coinbase"):
```

Comment:
Is this no longer necessary?

Reply:
hmmm, its from another email isnt? so i suppose its different transactions

Reply:
Why did you delete it?

Reply:
ohh, srry, my last comment was from mobile and didnt read carefully, i didnt delete it, just changed from "From coinbase" to "coinbaes earn". 
English phrase is "From coinbase Earn" and in spanish "De coinbase Earn".
I just changes startswith for endswith.

Reply:
i restore it. Now better?

Reply:
You can combine them all:
```
elif (
    transaction[_DETAILS][_SUBTITLE].startswith("From Coinbase") or
    transaction[_DETAILS][_SUBTITLE].endswith("Coinbase Earn") or
    transaction[_DESCRIPTION] == "Earn Task" or
):
```

There is a bug with the macos build we have to wait for. That is why the macos test is failing.